survey-api/src/main/java/org/kong/survey/dto/Question.java

```diff
@@ -21,13 +19,17 @@ public class Question {
     @Getter
     @Setter
     public static class Request {
+        private Integer questionId;
         private QuestionType questionType;
         private String question;
         private int order;
         List<SurveyAnswer.Request> answers;
     }
 
     @Getter
+    @Setter
+    @Builder
+    @NoArgsConstructor
```

Comment:
```suggestion
    @Setter
    @Builder
    @NoArgsConstructor    
```

각각의 어노테이션의 의도에 대해서 얘기해보도록 하죠.